.travis.yml

```diff
@@ -1,26 +1,19 @@
 ---
 language: python
-
-python:
-  - "2.7"
-
-# This (sudo: false) is needed to "run on container-based infrastructure" on
-# which cache: is available
-# http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+python: 2.7
 sudo: false
 
 cache: pip
 
 install:
-  - pip install PyYAML "pandas<0.19" "flake8<3.6.0" yamllint
-  - python setup.py install
+  - pip install -r requirements.txt
+  - pip install idr-utils/
```

Comment:
Interesting. What does this do?

Reply:
I think largely equivalent to `cd idr-utils && python setup.py install`. I remember there are cases where this form was preferable but I would need to dig up the threads.

Reply:
Agreed, but does it _do_ anything? i.e. if we're always using `python idr-utils/...` what's the purpose?

Reply:
Trying to remember, I might have been overoptimistic and expecting `pip install idr-utils/` to work out-of-the box  (requirements, scripts in the path). Getting it working properly obviously needs a bit more work :) I'll push it away